api/build.gradle

```diff
@@ -19,8 +19,9 @@ buildscript {
         OKHTTP_VERSION = '2.7.5'
         OPENCENSUS_VERSION = '0.31.1'
         SPRINGFOX_VERSION = '3.0.0'
-        SPRING_BOOT_VERSION = '2.7.4'
+        SPRING_BOOT_VERSION = '2.7.18'
         SPRING_FRAMEWORK_VERSION = '5.3.23'
+        SPRING_SECURITY_VERSION = '5.8.9'
```

Comment:
We will need to migrate to 6.x soon.  5.8.x is a transitional release to make that easier: https://docs.spring.io/spring-security/reference/5.8/migration/index.html